dm/dm/worker/task_checker.go

```diff
@@ -104,33 +104,13 @@ type TaskStatusChecker interface {
 // NewTaskStatusChecker is a TaskStatusChecker initializer.
 var NewTaskStatusChecker = NewRealTaskStatusChecker
 
-type backoffController struct {
-	// task name -> backoff counter
-	backoffs map[string]*backoff.Backoff
-
-	// task name -> task latest paused time that checker observes
-	latestPausedTime map[string]time.Time
-
-	// task name -> task latest block time, block means task paused with un-resumable error
-	latestBlockTime map[string]time.Time
-
-	// task name -> the latest auto resume time
-	latestResumeTime map[string]time.Time
-
-	latestRelayPausedTime time.Time
-	latestRelayBlockTime  time.Time
-	latestRelayResumeTime time.Time
-	relayBackoff          *backoff.Backoff
-}
-
-// newBackoffController returns a new backoffController instance.
-func newBackoffController() *backoffController {
-	return &backoffController{
-		backoffs:         make(map[string]*backoff.Backoff),
-		latestPausedTime: make(map[string]time.Time),
-		latestBlockTime:  make(map[string]time.Time),
-		latestResumeTime: make(map[string]time.Time),
-	}
+// AutoResumeTimes contains some Time and Backoff that are related to auto resume.
+// This structure is exposed for DM as library.
+type AutoResumeTimes struct {
```

Comment:
the suffix `Times` is confusing to me, especially used in variables like `subtaskTimes`, `relayTimes`, cannot see it's related to auto resume, may be `AutoResume[Info]` `subtaskAutoResume` `relayAutoResume`?